platform_versioned_docs/version-23.4.0/enterprise/prerequisites/azure.mdx

```diff
@@ -7,182 +7,310 @@ tags: [azure, prerequisites, configuration]
+    - **Subscription**: Select your Azure subscription.
 
-### Azure Resource Group
+    - **Resource group**: Enter a new resource group name (such as `towerrg`).
 
-1. Sign in to the [Azure portal](https://portal.azure.com).
+    - **Region**: Select the region where your assets will exist (such as `East US`).
 
-2. Select **Resource groups**.
+  1. Select **Review and Create**.
 
-3. Select **Add**.
+  1. Select **Create**.
 
-4. Enter the following values:
+</details>
+<details>
+   <summary>Create a resource group via Azure CLI</summary>
 
-   - **Subscription**: Select your Azure subscription.
+  Run the `az group create` command:
 
-   - **Resource group**: Enter a new resource group name (e.g., `towerrg`).
+   ```console
+  az group create --name $MY_RESOURCE_GROUP_NAME --location $REGION
+   ```
 
-   - **Region**: Select the region where your assets will exist (e.g., `East US`).
+</details>
 
-5. Select **Review and Create**.
+### Azure storage account
 
-6. Select **Create**.
+Create a storage account:
+- [Via the Azure portal][azure-storage-portal]
+- [Via the Azure CLI][azure-storage-cli]
 
-### Azure Storage Account
+<details>
+   <summary>Create a storage account via Azure portal</summary>
 
-1. Sign in to the [Azure portal](https://portal.azure.com).
+  1. Sign in to the [Azure portal](https://portal.azure.com).
 
-2. Select **Storage accounts**.
+  1. Select **Storage accounts**.
 
-3. Select **Create**.
+  1. Select **Create**.
 
-4. Enter the following values:
+  1. Enter the following values:
 
-   - **Subscription**: Select your Azure subscription.
+    - **Subscription**: Select your Azure subscription.
 
-   - **Resource group**: Enter your resource group name.
+    - **Resource group**: Enter your resource group name.
 
-   - **Storage account name**: Enter a new storage account name (e.g., `towerstorage`).
+    - **Storage account name**: Enter a new storage account name (e.g., `towerstorage`).
 
-   - **Region**: Select the region where your Resource Group exists (e.g., `East US`).
+    - **Region**: Select the region where your Resource Group exists (e.g., `East US`).
 
-   - **Performance**: Select `Standard`.
+    - **Performance**: Select `Standard`.
 
-   - **Redundancy**: Select `Geo-redundant storage (GRS)`.
+    - **Redundancy**: Select `Geo-redundant storage (GRS)`.
 
-5. Select **Review + create**. Note that the default values are used in the other tabs. See the [Azure documentation](https://docs.microsoft.com/en-ca/azure/storage/common/storage-account-create?tabs=azure-portal#create-a-storage-account-1) for further details on each setting.
+  1. Select **Review + create**. Note that the default values are used in the other tabs. See [Create a storage account][azure-storage] for further details on each setting.
```

Comment:
This link isn't rendering, it's showing up at markdown.

<img width="682" alt="Screenshot 2024-05-17 at 1 16 27 PM" src="https://github.com/seqeralabs/docs/assets/141646877/760ac485-0daf-4d83-aabb-bb2626ea4ce8">